src/robot/model/keyword.py

```diff
@@ -29,12 +34,15 @@ class Keyword(BodyItem):
     repr_args = ('name', 'args', 'assign')
     __slots__ = ['_name', 'args', 'assign', 'type']
 
-    def __init__(self, name='', args=(), assign=(), type=BodyItem.KEYWORD, parent=None):
+
+    def __init__(self, name:str='', args: tuple=(), assign: tuple=(), type=BodyItem.KEYWORD, parent=None):
```

Comment:
- `name:str=''` -> `name: str = ''`
- Should add type hints to all parameters.
- `args` and `assign` probably could also be lists, but I believe `tuple` is fine as a type hint. Can be changed to something more generic like `Sequence` later if needed.